ztp/gitops-subscriptions/argocd/example/acmpolicygenerator/kustomization.yaml

```diff
@@ -1,21 +1,31 @@
 generators:
 # This is common to all RAN deployments
 - acm-common-ranGen.yaml
-# This group policy is for all single-node deployments:
+# This group policy is for all single-node deployments. Use just one of the 2 below.
 - acm-group-du-sno-ranGen.yaml
+#- hub-side-templating/acm-group-du-sno-ranGen.yaml
```

Comment:
better to un-comment this line. We need the ci-job validate the change to template policies to avoid any changes that not recognized. Same for all template policies